.circleci/config.yml

```diff
@@ -9,15 +9,16 @@ jobs:
       # use `-browsers` prefix for selenium tests, e.g. `<image_name>-browsers`
 
       # Python
-      - image: circleci/python:3.7.4-buster
+      - image: circleci/python:3.7.8-buster
         environment:
           TZ: America/New_York
           SQLA_TEST_CONN: postgresql://postgres@0.0.0.0/cfdm_unit_test
 
       # PostgreSQL
-      - image: circleci/postgres:9.6.8
+      - image: circleci/postgres:10.11
         environment:
           POSTGRES_USER: postgres
+          POSTGRES_HOST_AUTH_METHOD: "trust"
```

Comment:
@pkfec thanks for your work on this! It's my understanding that this is just the setting for the test `postgres` database in Circle and won't impact our database settings.